pages/schedule/index.tsx

```diff
@@ -1,335 +1,194 @@
-      backgroundColor: `${grey[900]}`,
-      borderRadius: 8,
-      boxShadow: ` 0 0 16px 1px ${red[400]} `,
-    },
-    SponsorTypeAppointment: {
-      border: `2px solid ${orange[500]}`,
-      backgroundColor: `${grey[900]}`,
-      borderRadius: 8,
-      boxShadow: ` 0 0 16px 4px ${orange[500]} `,
-    },
-    TechTalkTypeAppointment: {
-      border: `2px solid ${indigo[500]}`,
-      backgroundColor: `${grey[900]}`,
-      borderRadius: 8,
-      boxShadow: ` 0 0 16px 4px ${indigo[500]} `,
-    },
-    WorkshopTypeAppointment: {
-      border: `2px solid ${purple[500]}`,
-      backgroundColor: `${grey[900]}`,
-      borderRadius: 8,
-      boxShadow: ` 0 0 16px 4px ${purple[500]} `,
-    },
-    SocialTypeAppointment: {
-      border: `2px solid ${blue[500]}`,
-      backgroundColor: `${grey[900]}`,
-      borderRadius: 8,
-      boxShadow: ` 0 0 16px 4px ${blue[500]} `,
-    },
-    weekEndCell: {
-      backgroundColor: alpha(palette.action.disabledBackground, 0.04),
-      '&:hover': {
-        backgroundColor: alpha(palette.action.disabledBackground, 0.04),
-      },
-      '&:focus': {
-        backgroundColor: alpha(palette.action.disabledBackground, 0.04),
-      },
-    },
-    weekEndDayScaleCell: {
-      backgroundColor: alpha(palette.action.disabledBackground, 0.06),
-    },
-    text: {
-      overflow: 'hidden',
-      textOverflow: 'ellipsis',
-      whiteSpace: 'nowrap',
-    },
-    content: {
-      opacity: 0.7,
-    },
-    container: {
-      width: '100%',
-      lineHeight: 1.2,
-      height: '100%',
-    },
-  });
-
-type AppointmentProps = Appointments.AppointmentProps & WithStyles<typeof styles>;
-type AppointmentContentProps = Appointments.AppointmentContentProps & WithStyles<typeof styles>;
-
-const isWeekEnd = (date: Date): boolean => date.getDay() === 0 || date.getDay() === 6;
-const defaultCurrentDate = new Date(2021, 10, 13, 9, 0);
-{
-  /* !!!change */
-}
-
-const AppointmentContent = withStyles(styles, { name: 'AppointmentContent' })(
-  ({ classes, data, ...restProps }: AppointmentContentProps) => {
-    let Event = 'Event';
-    if (data.Event === 2) Event = 'Sponsor';
-    if (data.Event === 3) Event = 'Tech Talk';
-    if (data.Event === 4) Event = 'Workshop';
-    if (data.Event === 5) Event = 'Social';
-
-    return (
-      <Appointments.AppointmentContent {...restProps} data={data}>
-        <div className={classes.container}>
-          <div className={classes.text}>{data.title}</div>
-          <div className={classNames(classes.text, classes.content)}>{`Type: ${Event}`}</div>
-          <div className={classNames(classes.text, classes.content)}>
-            {`Location: ${data.location}`}
-          </div>
-        </div>
-      </Appointments.AppointmentContent>
-    );
-  },
-);
+/* Set event dates and times */
+const day1StartDateAndTime = new Date(2024, 4, 25, 9, 0);
+const day2StartDateAndTime = new Date(2024, 4, 26, 9, 0);
+const eventEndDateAndTime = new Date(2024, 4, 27, 9, 0);
```

Comment:
Is it possible to somehow have these values stored in database and fetch when needed?

Reply:
@DedsecKnight Okay, how does it sound if I let users input the days of their hackathon somewhere in the admin side, store that data in the firebase database, and then pull that data here?

Reply:
Sounds reasonable to me, feel free to give it a try :+1: